swift/extractor/infra/SwiftDispatcher.h

```diff
@@ -151,7 +152,13 @@ class SwiftDispatcher {
       return *l;
     }
     waitingForNewLabel = e;
+    // TODO: more generic and informational visiting one-line log
+    if constexpr (std::is_convertible_v<E, const swift::ValueDecl*>) {
+      const swift::ValueDecl* x = e;
+      LOG_TRACE("{}", x->getName().getBaseIdentifier().str());
```

Comment:
definitely, though I think I will be moving this kind of tracing to the translators, where I also have programmatic access to the swift entity type names because of macro metaprogramming. But I'll leave it for a PR down the road, and for now remove this draft log